### PR TITLE
Support Environment per VPCs in one account

### LINKED ISSUE #71: Support multiple Consul clusters per account

### PR DESCRIPTION
**Pull request by gozer**
Fixes #71

### CODE REVIEW

File: nubis/terraform/main.tf [hunk 1]
```diff
@@ -39,13 +52,13 @@ resource "aws_autoscaling_group" "consul" {
...
     key = "Name"
-    value = "Consul member node (v/${var.release}.${var.build})"
+    value = "Consul server node (v/${var.release}.${var.build}) for ${var.service_name} in ${var.environment}"
     propagate_at_launch = true
   }
 }
 
 resource "aws_security_group" "consul" {
-  name = "${var.project}"
+  name =   "${var.project}-${var.environment}"
```
Comment (tinnightcap) [inline nubis/terraform/main.tf]: extra whitespace, I assume non-breaking?

**Comment by tinnightcap** [comment]
r=jd

File: nubis/terraform/main.tf (hunk 1 above)
Comment (gozer) [inline nubis/terraform/main.tf]: Yeah, non-breaking, but ugly

Comment (gozer) [inline nubis/terraform/main.tf]: Sneaky, but fixed in ride-along of #75

**Pull request merged**